frontend_tests/node_tests/stream_data.js

```diff
@@ -238,6 +238,10 @@ zrequire('stream_data');
     ok = stream_data.add_subscriber('UNKNOWN', brutus.user_id);
     assert(!ok);
 
+    // Same as above but for removing subscribers
+    ok = stream_data.remove_subscriber('UNKNOWN', brutus.user_id);
```

Comment:
Before this line, you should redefine `blueslip.warn` to match the warning message: "We got a remove_subscriber call for a non-existent stream " + stream_name.